app/models/ability.rb

```diff
@@ -0,0 +1,18 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    if user.present?
+      can %i(create read update destroy), Journal
+      can %i(create read update destroy), JournalEntry
+      can %i(create show new), User
+      can :update, User, id: user.id
+      cannot :index, User
+      if user.admin?
+        can :manage, :all
+      end
+    else
+      can %i(create show new), User
```

Comment:
which user is this? Let's keep the ability readable

Reply:
I modified it but it wont work with return unless NO_user because everyone needs some rights